tests/UnifiedSpecTests/gridfs/upload.json

```diff
@@ -1,616 +0,0 @@
-{
```

Comment:
I assume this was unintentionally deleted, as `upload-disableMD5.json` also uses deprecated fields and wasn't touched.